bucket/terraform.json

```
@@ -2,7 +2,7 @@
     "version": "1.6.1",
     "description": "Create and manage infrastructure as code using simple declarative configuration files.",
     "homepage": "https://www.terraform.io",
-    "license": "MPL-2.0",
+    "license": "BSL-1.1",
```

```suggestion
    "license": "BUSL-1.1",
```
The SPDX identifier has 'U' in it: https://spdx.org/licenses/BUSL-1.1.html